src/properties/triangles.c

```diff
@@ -668,8 +664,8 @@ int igraph_transitivity_undirected(const igraph_t *graph,
     IGRAPH_FINALLY(igraph_adjlist_destroy, &allneis);
 
     neis = IGRAPH_CALLOC(no_of_nodes, long int);
-    if (neis == 0) {
-        IGRAPH_ERROR("undirected transitivity failed", IGRAPH_ENOMEM);
+    if (! neis) {
+        IGRAPH_ERROR("Insufficient memory for undirected global transitivity.", IGRAPH_ENOMEM);
```

Comment:
Same here ("Not enough memory" is already appended to the string in the error handler).